bin/next

```diff
@@ -5,6 +5,24 @@ import { spawn } from 'cross-spawn'
 import { watchFile } from 'fs'
 import pkg from '../../package.json'
 
+if (pkg.peerDependencies) {
+  let peerDependencyUnavailable = false
+
+  Object.keys(pkg.peerDependencies).forEach(dependency => {
+    try {
+      // When 'npm link' is used it checks the clone location. Not the project.
+      require.resolve(dependency)
+    } catch (err) {
+      console.log(`${dependency} not found. Please install ${dependency} using 'npm install ${dependency}'`)
+      peerDependencyUnavailable = true
+    }
+  })
+
+  if (peerDependencyUnavailable) {
```

Comment:
@sakulstra the idea behind is to make that usecase even better precisely, by not being opinionated about what flavor of React (fiber vs non-fiber) we bundle, or even enabling alt-Reacts.

At the moment, however, we need to figure out how we do an alias for the Node side of things, so that we use the server-renderer of each package.

We might introduce `alias` in `next.config.js` for this purpose (see https://github.com/zeit/next.js/pull/767), but I wish we could avoid it. Maybe we need to figure out a better way to run webpack on the server code. cc @nkzawa @arunoda